contracts/IDataFeed.sol

```diff
@@ -2,8 +2,9 @@ pragma solidity >=0.4.24;
 
 import "tidbit/contracts/DataFeedOracles/DataFeedOracleBase.sol";
 
-interface ITidbitDataFeedOracle {
+interface IDataFeed {
   function setResult(DataFeedOracleBase[] memory _dataFeeds) public;
```

Comment:
does this need both `setResult` and `logResult`? maybe `setResult` can be removed now?

Reply:
yah that's true. It also feels a big weird to have the Interface in this repo, that datafeeds outside the repo need to follow. Fine for now, but there's something about this that feels a bit....off to me right now